src/renderer/components/+events/event.store.ts

```diff
@@ -20,8 +20,8 @@ export class EventStore extends KubeObjectStore<KubeEvent> {
 
   protected sortItems(items: KubeEvent[]) {
     return super.sortItems(items, [
-      event => event.metadata.creationTimestamp
-    ], "desc");
+      event => event.getTimeDiffFromNow(), // keep events order as timeline ("fresh" on top)
+    ], "asc");
```

Comment:
Why it's better than sorting by `creationTimestamp`? Other places still use `creationTimestamp` for sorting by age. Pods, for example 
```
[columnId.age]: (pod: Pod) => pod.metadata.creationTimestamp
```

Reply:
Sorting is broken with raw `creationTimestamp` (it's date string, not timestamp)